### DISCUSSION TITLE
Add options to the protobuf processor to indent and emit unpopulated fields

### DISCUSSION THREAD

**Discussion by jkunnen-structurizit** [redpanda-data/connect#2419] [Ideas]
At this moment no indentation and no emit of unpopulated fields is possible in the protobuf processor when marshaling to json ouput.
Especially the option to emit unpopulated fields is needed.

**Comment by mihaitodor** [comment]
Hi @jkunnen-structurizit, can you please offer more details about this? I think the indentation issue can be solved using the [`format_json()` bloblang method](https://www.benthos.dev/docs/guides/bloblang/methods#format_json) in a [`mapping` processor](https://www.benthos.dev/docs/components/processors/mapping), but I'm not sure what you mean by emitting unpopulated fields. Can you provide a simple example that I can run myself?
